.travis.yml

```diff
@@ -2,31 +2,34 @@
 sudo: false
 language: go
 go:
-- 1.10.x
+  - 1.14.x
 go_import_path: github.com/elastic/go-libaudit
 
 install:
-- go get -u github.com/elastic/go-licenser
-- go get -d -t ./...
+  - GO111MODULE=off go get -u github.com/elastic/go-licenser
+  - go get -d -t ./...
 
 script:
-- go-licenser -d
-- >
-  find . -name '*.go' | grep -v vendor | xargs gofmt -s -l | read &&
-  echo "Code differs from gofmt's style. Run 'gofmt -s -w .'" 1>&2 && exit 1 || true
-- go test -v $(go list ./... | grep -v /vendor/)
-- GOARCH=386 go test -v $(go list ./... | grep -v /vendor/)
-- mkdir -p build/bin
-- go build -o build/bin/audit ./cmd/audit/
-- go build -o build/bin/auparse ./cmd/auparse/
+  - go mod download
+  - go mod verify
+  - go mod tidy && [ -z "$(git status --porcelain go.mod go.sum)" ] || (echo "Go module manifest changed. Run 'go mod tidy'" 1>&2 && exit 1)
+  - go-licenser -d
+  - >
+    find . -name '*.go' | grep -v vendor | xargs gofmt -s -l | read &&
+    echo "Code differs from gofmt's style. Run 'gofmt -s -w .'" 1>&2 && exit 1 || true
+  - go test -v $(go list ./... | grep -v /vendor/)
```

Comment:
Not a new issue so you don't have to change it if you don't want, but the vendor filtering hasn't been needed for a while (there's no `vendor/` code and I think go tools skip vendor when using `./...` now).